eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/model/SubSnpNoHgvs.java

```diff
@@ -195,4 +245,123 @@ public boolean isAssemblyMatch() {
     public void setAssemblyMatch(boolean assemblyMatch) {
         this.assemblyMatch = assemblyMatch;
     }
+
+    public String getReferenceInForwardStrand() {
+        if (contigOrientation.equals(Orientation.REVERSE)) {
```

Comment:
Maybe we should also check `snpOrientation` here? I don't know for sure, but it is displayed along the `contigOrientation` in the table, see [this example](https://www.ncbi.nlm.nih.gov/projects/SNP/snp_ref.cgi?rs=rs666).